week-3.md

```diff
@@ -0,0 +1,82 @@
+
+One thing that's been working out really well (in my opinion, anyway!) is how engaged reviewers have been with the regular process of filing issues and pull-requests on github to make improvements to the drafts. Since that's the usual way of improving most open source Bitcoin related things, we've been encouraging people to focus on that, rather than collecting review results via google forms or similar. But to help avoid things getting missed, we've added some free form fields to the knowledge check survey for capturing things that might need changes that haven't been filed in github.
+
+## Week 1 Feedback
+
+We got 36 responses (about 20%) to the "week 1 feedback" survey -- summary stats were:
+
+ * Solo review: 2 hours (15 responses), 3+ hours (11 responses), 1 hour (6), 30 min (4)
+ * Study group: 1 hour (13), 2 hours (8), None! (8), 30 minutes (6), 3+ hours (1)
+ * Q&A session: 1 hour (19), 30 minutes (7), None! (5), 2 hours (4), 3+ hours (1)
+ * PoC work: None! (33), 3+ hours (1), 1 hour (1), 30 minutes (1)
+
+Most people rated their first week progress as 2 or 3, so about as expected, and resources as 3-5 which is great. Study group rating was bit more spread out, with nine responses give a low rating of 1 or 2, twenty giving mid-good rating of 3 or 4, and six giving top marks of 5. So if your group is doing particularly well, it might be worth writing a blog post or making a youtube video or similar about what's going right so others can learn from you; alternatively if your group is not doing so great, it might be interesting to lurk through some of the other groups' slack channels to see what works for them and perhaps try it in your group.
+
+This poll was just for the first week, so it's already out of date, but since this is all an experiment, might be of interest anyway.
+
+## Results from Week Two
+
+We've had the second two Q&A sessions, with meetbot logs: [week 2 Q&A 1](), [week 2 Q&A 2]() -- but as it turns out the second Q&A didn't actually have any questions. Since Q&A 2 in week one seemed a bit sparser too, we'll just advertise the single slot this week, but if that's not convenient for you, do feel free to ask questions anytime -- experts are lurking in that channel most of the time, and even if they're not available to answer immediately, if you can stay connected you will often get an answer sometime later. (If you can't stay connected, you can use the [logs](http://www.erisian.com.au/taproot-bip-review/) to catch your answer, but it might be wise to mention that you'll be doing so to avoid people thinking there's no point responding because you've left the channel)
```

Comment:
http://www.erisian.com.au/meetbot/taproot-bip-review/2019/taproot-bip-review.2019-11-12-19.00.log.html
and 
http://www.erisian.com.au/meetbot/taproot-bip-review/2019/taproot-bip-review.2019-11-14-02.00.log.html